ui/raidboss/data/05-shb/trial/varis-ex.txt

```diff
@@ -0,0 +1,116 @@
+# Memoria Misera Extreme
+# -ic "Phantom Varis" -ii 4CEF 4CF1 4CDD 4CEE
```

Comment:
Could you go through and double-check all the ability IDs that you removed from the timeline? It's easy to miss some when making manual edits to individual lines!

(If they are used at least once somewhere, they do not have to be present in the list of ignored IDs, but if they appear in the log when running `make_timeline` AND are not anywhere in the timeline file, they should be listed here.)